source/fundamentals/data-formats/codecs.txt

```diff
@@ -39,4 +39,11 @@ BsonTypeClassMap
 Custom Codec Example
 --------------------
 
-- Example, perhaps something like this https://stackoverflow.com/questions/45083885/decode-document-to-a-java-class-using-custom-mongo-codec
\ No newline at end of file
+- Methods to override
+- Encode: Describe parameters: BsonReader, EncoderContext
+- Decode: Describe parameters: BsonReader, DecoderContext
+- Example, perhaps something like this https://stackoverflow.com/questions/45083885/decode-document-to-a-java-class-using-custom-mongo-codec
+- Sample results from custom codec example
+- Explain one or more commonly-encountered exceptions such as BsonSerializationException
```

Comment:
Not sure if this is necessary.  We don't typically catch those exceptions in our own codecs

Reply:
Did you remove this?

Reply:
From both my last commit, and the preview I did remove it: https://github.com/mongodb/docs-java/pull/35/commits/583e2c87430e3f3e2c43b2fcea9d889b085d60da#diff-dda4e6ecdf7c59166b009c08afae23ebL47

Do you think I should bring it back?

Reply:
No, I was just expecting a "Done" comment to indicate it.